stock-tracker/src/features/chart/redux/chartMiddleware.ts

```diff
@@ -1,9 +1,12 @@
 import { SET_CHART_TIME } from "./actionTypes";
 import { CHART_TIME } from "../../../socket/eventTypes";
+import { Middleware } from "redux";
+import { AppState } from "../../../store/rootReducer";
+import { SocketDependency } from "../../../utils/socketService";
```

Comment:
nit: Might be nice to make an `index.js` file and export your `SocketDependancy` to it, so your file path becomes:
`import { SocketDependency } from "../../../utils";`